### PR TITLE
Feat/email components alert

### PR DESCRIPTION
**Pull request by chadimaOndrej**

### CODE REVIEW

File: packages/email-components/src/components/Alert/Alert.scss [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    vertical-align: middle;
+    border: theme.$icon-with-border-border;
+    border-radius: theme.$icon-with-border-radius;
+}
+
+.Alert__content {
+    padding: theme.$padding;
+}
+
+.Alert--center {
```
Comment (vmilersky) [inline packages/email-components/src/components/Alert/Alert.scss:45]: Modifiers of main Block element should come after the main Block styles, in this case after line 8.

File: packages/email-components/src/scss/tokens/_typography.scss [hunk 2]
```diff
@@ -27,4 +27,4 @@ $font-weight-normal: 400;
 $font-weight-bold: 700;
 
 // Font families
-$font-family-base: arial, sans-serif;
+$font-family-base: arial, helvetica, sans-serif;
```
Comment (vmilersky) [inline packages/email-components/src/scss/tokens/_typography.scss:30]: This change is something new? I thought it'll be OK to use values from DS tokens - we agreed on this last week - Arial and sans-serif should do just fine.

I see a lot of inconsistencies - see screenshot (not only the highlighted line).

Screenshot 2023-05-22 at 14 34 30

File: packages/email-components/src/templates/styleGuide.mjml [hunk 3]
```diff
@@ -95,5 +95,26 @@
...
+    <mjc-content>
+      <mj-column width="100%">
+        <mj-text><strong>COMPONENT: Alert</strong></mj-text>
+        <mj-spacer height="10px" />
+        <mjc-alert>Toto je <strong>Success</strong> alert.</mjc-alert>
+        <mj-spacer height="10px" />
+        <mjc-alert color="informative">Toto je informative alert <strong>s dulezitym sdelenim</strong></mjc-alert>
+        <mj-spacer height="10px" />
+        <mjc-alert color="warning"
+          >Toto je warning alert <strong>s dulezitym sdelenim</strong>. A tento text je dlouhy, proto se bude muset
```
Comment (vmilersky) [inline packages/email-components/src/templates/styleGuide.mjml:108]: Some weird new lines perhaps?

File: packages/email-components/src/components/Button/README.md [hunk 4]
```diff
@@ -11,7 +11,7 @@ The `mjc-button` component is button for e-mail template.
 | Prop name        | Type                      | Default   | Required | Description     |
 | ---------------- | ------------------------- | --------- | -------- | --------------- |
 | `align`          | `left \| right \| center` | `left`    | no       | Button position |
-| `color`          | `primary \| secondary`    | `primary` | yes      | Url to image    |
+| `color`          | `primary \| secondary`    | `primary` | yes      | Button color    |
 | `href`           | `string`                  | -         | no       | Link to         |
```
Comment (vmilersky) [inline packages/email-components/src/components/Button/README.md:15]: Link to what? Please provide subject.

File: packages/email-components/src/scss/tokens/_typography.scss (hunk 2 above)
Comment (chadimaOndrej) [reply packages/email-components/src/scss/tokens/_typography.scss:30]: Thanks for this check, now it is consistent.

**Pull request merged**